README.md

```diff
@@ -68,12 +68,12 @@ Keep in mind that you have to be connected to the VPN for this to work, even in
 
 Sometimes the default development proxy started with `yarn start||local` can be
 slow and unstable, which can be problematic while testing. To overcome this you
-can use the [consoledot-testing-proxy](https://github.com/dvagner/consoledot-testing-proxy)
+can use the [frontend-development-proxy](https://github.com/RedHatInsights/frontend-development-proxy)
 and run the app in static mode.
 
 1. `yarn fec static` to start the app in static mode
 
-2. `podman run -d -e HTTPS_PROXY=$RH_PROXY_URL -p 1337:1337 -v "$(pwd)/config:/config:ro,Z" --replace --name consoledot-testing-proxy quay.io/dvagner/consoledot-testing-proxy` to run the proxy against stage
+2. `podman run -d -e HTTPS_PROXY=$RH_PROXY_URL -p 1337:1337 -v "$(pwd)/config:/config:ro,Z" --replace --name frontend-development-proxy quay.io/redhat-user-workloads/hcc-platex-services-tenant/frontend-development-proxy:latest` to run the proxy against stage
```

Comment:
thanks for updating the CI to this new container! 

sort of a sidenote: I think this entire readme section could be removed as it was useful in the period after the proxy was created, but before it was integrated into the `fec` as the `dev-proxy` command 💭 
now this isn't necessary anymore (only in CodeBuild CI with `--network=host`, because of the D-in-D setup there) 😅

Reply:
I think it is better to leave it as this is the only place where it actually described on how to run it this way and we're using it during the CI. It was helpful to me to understand what we are doing in CI as it is different from local run.